internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -450,7 +450,12 @@ func (r *Reconciler) reconcileCluster(ctx context.Context, s *scope.Scope) error
 		return nil
 	}
 
-	log.Infof("Patching %s", tlog.KObj{Obj: s.Current.Cluster})
+	changes := patchHelper.Changes()
```

Comment:
has there been consideration to let this changes log be part of the patchHelper.Patch func?